### ISSUE TITLE
Trouble Reproducing DINOv2 Depth Estimation results on NYUv2 (also, SyncBN?)

### ISSUE THREAD

**Issue by anonymous123-user** [facebookresearch/dinov2#564]
Hi! I am trying to reproduce the DINOv2(ViT-S) depth estimation results on NYUv2, but I'm constantly getting about 0.1 RMSE higher than reported in the paper. I'm trying to do this using the pretrained ViT-S and the publicly available prediction head (lin-1) (so no training of my own).

Additionally, is there a batch norm used? I noticed that decode_heads.py has "BNHead", which has the "self.bn = nn.SyncBatchNorm(self.in_channels)" commented out, whereas in DINOv3, the paper mentions that a batch norm is used.
